Integrations/QRadar/QRadar.yml

```diff
@@ -1234,8 +1234,8 @@ script:
         Epoch January 1st 1970). If ''date_value'' is set to ''True'', then the argument
         will be converted from date in format: ''%Y-%m-%dT%H:%M:%S.%f000Z'' (e.g.
         ''2018-11-06T08:56:41.000000Z'') to epoch.'
-      isArray: false
-      name: value
+      isArray: true
+      name: values
```

Comment:
you can't change this name, it will break BC for customers

Reply:
done